src/app/components/footer/footer-links/footer-links.component.scss

```diff
@@ -0,0 +1,62 @@
+@use "./src/assets/scss/variables" as variable;
+@use "./src/assets/scss/mixins" as mixin;
+@use "./src/assets/scss/functions" as function;
+
+:host {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  min-width: 75%;
```

Comment:
I have made this block's children min-width:23%. If any list size will be more then 23% then it's width will grow up. So, with the grow of child tags will grow host size also till 100% and it will drop down in the next row.